input/templates/partials/myaccount/my-address-book-address-form.hbs

```diff
@@ -29,7 +29,7 @@
     <div class="input-block">
       <label for="billing-checkbox">
         <span>
-          <input id="default-billing-checkbox" type="checkbox" name="defaultBillingAddress">
+          <input id="default-billing-checkbox" type="checkbox" name="defaultBillingAddress" {{#if form.defaultBillingAddress}}checked{{/if}}>
```

Comment:
We have to check whether the form has the value `defaultBillingAddress` to `true` to then check the checkbox, otherwise this checkbox would always be disabled.